docs/src/main/asciidoc/_high-availability/jgroups/External_Protocols.adoc

```diff
@@ -0,0 +1,108 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+=== Adding an External Module
+
+Any JGroups protocol implementing `org.jgroups.stack.Protocol` can be added to the stack.
+While WildFly provides multitude of JGroups protocols out-of-box,
+users sometimes require addition of external protocols.
+We also manage a GitHub organization named https://github.com/jgroups-extras[jgroups-extras] where we host source code
```

Comment:
Who is "We"?

WildFly? (the project)
JGroups? (the project)

If it's not the WildFly project, don't use 'We' at the beginning here.